### ISSUE TITLE
when i use mask rcnn to detection,result is false

### ISSUE THREAD

**Issue by nzmyaaa** [open-mmlab/mmdetection#8486]
config_file='../../mmdetection/configs/mask_rcnn/mask_rcnn_r50_caffe_fpn_mstrain-poly_3x_coco.py'
#config_file='../../mmdetection/configs/faster_rcnn/faster_rcnn_r50_caffe_fpn_mstrain_3x_coco.py'
#config_file='../../mmdetection/configs/yolo/yolov3_d53_mstrain-416_273e_coco.py'
checkpoint_file='../mmdetection/checkpoints/mask_rcnn_r50_caffe_fpn_mstrain-poly_3x_coco_bbox_mAP-0.408__segm_mAP-0.37_20200504_163245-42aa3d00.pth'
#checkpoint_file='../mmdetection/checkpoints/faster_rcnn_r50_caffe_fpn_mstrain_3x_coco_20210526_095054-1f77628b.pth'
model=init_detector(config_file,checkpoint_file,device='cuda:0')
#in_folder='../mmdetection/rumor_images/'
in_folder='../mmdetection/nonrumor_images/'
#out_folder='../mmdetection/fenge_rumor_images/'
#out_folder='../mmdetection/newsplit_rumor_images/'
out_folder='../mmdetection/fengenonrumor_images/'

**Comment by jbwang1997** [comment]
Hello @nzmyaaa, could you provide the error report.

**Comment by nzmyaaa** [comment]
when i use the mmdetection to segment the instance image,my model is mask rcnn,and print the result,some result is false.
 
this is my code.
import pickle

import torch
from mmdet.apis import init_detector,inference_detector,show_result_pyplot
import numpy as np
import os
import cv2
import random
import mmcv
from torch.utils.data import Dataset, DataLoader
import argparse
#import process_data_weibo as process_data
import copy

config_file='../../mmdetection/configs/mask_rcnn/mask_rcnn_r50_caffe_fpn_mstrain-poly_3x_coco.py'
#config_file='../../mmdetection/configs/faster_rcnn/faster_rcnn_r50_caffe_fpn_mstrain_3x_coco.py'
#config_file='../../mmdetection/configs/yolo/yolov3_d53_mstrain-416_273e_coco.py'
checkpoint_file='../mmdetection/checkpoints/mask_rcnn_r50_caffe_fpn_mstrain-poly_3x_coco_bbox_mAP-0.408__segm_mAP-0.37_20200504_163245-42aa3d00.pth'
#checkpoint_file='../mmdetection/checkpoints/faster_rcnn_r50_caffe_fpn_mstrain_3x_coco_20210526_095054-1f77628b.pth'
model=init_detector(config_file,checkpoint_file,device='cuda:0')
#in_folder='../mmdetection/rumor_images/'
in_folder='../mmdetection/nonrumor_images/'
#out_folder='../mmdetection/fenge_rumor_images/'
#out_folder='../mmdetection/newsplit_rumor_images/'
out_folder='../mmdetection/fengenonrumor_images/'


results=[]
leng=0
i=0

# new fangfa
for image in os.listdir(in_folder):
    leng=leng+1
    #print(image)
print(leng)

for file_name in (os.listdir(in_folder)):


     img_path=os.path.join(in_folder,file_name)
     img=cv2.imread(img_path)
     #img=np.asarray(img)
     #print(img)
     #print(type(img))
     #print(img.shape)
     try:
        result=inference_detector(model,img)
        result=list(result)
        #result=torch.as_tensor(result)
        print(result)
        #print(type(result))
       # print(result.size())
        save_path=os.path.join(out_folder,file_name)
        model.show_result(img,result,out_file=save_path)
     #if(result.item()==False):
         #result=torch.ones_like(result.item())

        results.append(result)

result is:
[False,False,False,,,,False],[],[],[]

can i know why the result is flase
